tardis/visualization/tools/tests/test_sdec_plot.py

```diff
@@ -127,6 +157,7 @@ def plotter(self, simulation_simple):
         -------
         tardis.visualization.tools.sdec_plot.SDECPlotter
         """
+        # request.cls.regression_data = RegressionData(request)
```

Comment:
delete?

Reply:
Fixed in [`257931c` (#2785)](https://github.com/tardis-sn/tardis/pull/2785/commits/257931c30df41e9a13544c0949e0d686f510a5d3)